contributor_docs/sidebar.md

```diff
@@ -8,6 +8,7 @@
   - [Discussions](discussions.md)
   - [Design Principles](design_principles.md)
   - [Roadmap](roadmap.md)
+  - [Contribution Organization](organization.md)
```

Comment:
I think this title is ambiguous because of the multiple meanings of organization. Maybe a slightly more verbose title could help people know whether to click?